pkg/agent/core/state_test.go

```diff
@@ -202,19 +210,24 @@ func getDesiredResources(state *core.State, now time.Time) api.Resources {
 	return res
 }
 
-func doInitialPluginRequest(
-	a helpers.Assert,
-	state *core.State,
-	clock *helpers.FakeClock,
-	requestTime time.Duration,
-	metrics *api.Metrics,
-	resources api.Resources,
-) {
+func logicalTime(clock *helpers.FakeClock, value int64) *vmv1.LogicalTime {
+	return &vmv1.LogicalTime{
+		Value:     value,
+		UpdatedAt: v1.NewTime(clock.Now()),
+	}
+}
+
+func doInitialPluginRequest(a helpers.Assert, state *core.State, clock *helpers.FakeClock, requestTime time.Duration, metrics *api.Metrics, resources api.Resources, enableLogicalClock bool) {
```

Comment:
nit: very long line

(WDYT about adding a lint for this?)

Reply:
Lint 👍 , added to #921

Reply:
(line is still very long)